git/functions/gdel

```diff
@@ -3,7 +3,14 @@
 # delete given branch locally & remotely
 
 [[ -z $1 ]] && {
-  echo 'No branch given, aborting'
+  echo 'error: the following arguments were not provided'
+  echo ''
+  echo '  BRANCH-NAME'
+  echo ''
+  echo 'USAGE:'
+  echo ''
+  echo '    gdel BRANCH-NAME'
+  echo ''
```

Comment:
[Here docs](https://en.wikipedia.org/wiki/Here_document#Unix_shells) can make this a little nicer. Would like below:

> Note: spacing is a little weird inside Here docs, all whitespace/indentation choices below are purposeful

```bash
[[ -z $1 ]] && {
  cat << EOF
error: the following arguments were not provided

  BRANCH-NAME

USAGE:

  gdel BRANCH-NAME
EOF
  return 1
}
```